frontend/next-i18next.config.js

```diff
@@ -2,9 +2,10 @@
 
 module.exports = {
   i18n: {
-    defaultLocale: "en",
+    defaultLocale: "en_GB",
```

Comment:
GB as default! :tada:

Reply:
Yeah, looking at the formats, I think most of the world will have a much better time with those number/date formats.

Reply:
Probably won't last, if we go for the less invasive version for now.